### ISSUE TITLE
Creating 3300 pooled addresses cause creating 5 broker pods

### ISSUE THREAD

**Issue by tkratky** [EnMasseProject/enmasse#1306]
Description:
When 3300 pooled addresses with self defined plan pooled-xxs-queue (consume 0.001 broker resource) then 5 broker pods is created. This issue has no impact on functionality. Everything works fine.
```
{
    "apiVersion": "v1",
    "data": {
        "definition": "{\"apiVersion\":\"enmasse.io/v1\",\"kind\":\"AddressPlan\",\"metadata\":{\"name\":\"pooled-xxs-queue\"},\"displayName\":\"pooled-xxs-queue\",\"displayOrder\":\"0\",\"shortDescription\":\"Newly defined address plan.\",\"longDescription\":\"Newly defined address plan.\",\"uuid\":\"f64cc30e-0d9e-11e8-ba89-0ed5f89f718b\",\"addressType\":\"queue\",\"requiredResources\":[{\"name\":\"broker\",\"credit\":0.001}]}"
    },
    "kind": "ConfigMap",
    "metadata": {
        "creationTimestamp": "2018-06-01T07:47:23Z",
        "labels": {
            "type": "address-plan"
        },
        "name": "address-plan-pooled-xxs-queue",
        "namespace": "many-addresses-standard",
        "resourceVersion": "118833",
        "selfLink": "/api/v1/namespaces/many-addresses-standard/configmaps/address-plan-pooled-xxs-queue",
        "uid": "05507dfb-6570-11e8-8698-525400e26b8e"
    }
}
```

reproducible via automated marathon test in enable-tests branch: https://github.com/EnMasseProject/enmasse/blob/9fffde3d97a989a55d6311c9769c1bb647fa65c1/systemtests/src/test/java/io/enmasse/systemtest/marathon/PlansMarathonTest.java#L41

test creates addresses one by one, so it takes a lot of time

in standard controller log you can see cause of 5 pods:
broker2 usage:  `{broker={broker-2={used=1.0000000000000007, needed=2}`
```
2018-06-01 10:09:55 INFO AddressController:121 - Usage: {broker={broker-2={used=1.0000000000000007, needed=2}, broker-3={used=0.23500000000000018, needed=1}, broker-0={used=0.9990000000000008, needed=1}, broker-1={used=0.9990000000000008, needed=1}}}, Needed: {broker={broker-2={used=1.0000000000000007, needed=2}, broker-3={used=0.23500000000000018, needed=1}, broker-0={used=0.9990000000000008, needed=1}, broker-1={used=0.9990000000000008, needed=1}}}
```

**Comment by lulf** [comment]
I'm wondering if this is a consequence of adding doubles together and always rounding up. Wondering if just rounding to a fixed number of decimals is sufficient to fix this.

**Comment by lulf** [comment]
Probably same issue as #2042, fix in #2086

**Comment by lulf** [comment]
Fixed in #2101
